ReactNativeClient/lib/share.js

```diff
@@ -0,0 +1,13 @@
+const { NativeModules, Platform } = require('react-native');
```

Comment:
Please name the file ShareExtension.js, as you import it under this name (also could you convert it to TypeScript please?)